carma-messenger-ui/website/widgets/emergencyResponse/widget.js

```diff
@@ -113,10 +167,9 @@ var service_get_emergency_route = () => {
     });
 
     get_emergency_route.callService(request, function (result) {
```

Comment:
I think we may be able to remove this now (along with all other usages of `get_emergency_route`) since the Web UI widget will no longer be displaying the route name.

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/pull/177/commits/89771b758699a93896e5555ceb112d4a9e320309